spotify_project/base/templates/index.html

```diff
@@ -2,15 +2,16 @@
 {% block title %}Home{% endblock title %}
 {% block content %}
     <div class="text-center">
-        <h1>Welcome</h1>
+        <h1>Welcome {{ user.username }}</h1>
```

Comment:
This is a little cheeky. If the user is not defined, you'll get a invalid template variable. Which Django renders as an empty string. Unless you tell it to do something different. Technically speaking, the inclusion of `{{ user.username }}` ought to be wrapped in `{% if user.is_authenticated %}`

Reply:
Oh, that makes sense. I was including it mostly as a debugging tool for the user situation, but that's a good thing to point out. I was just satisfied with it rendering correctly.